joomla/joomla-cms
# [4.0] Move toolbar apply items to thead (com_content)

Pull Request for Issue #16000  .

### Summary of Changes
Second of 2 PRs to apply #16000. Moves the toolbar apply actions to the table header in com_content->articles.

### Testing Instructions
Apply patch and navigate to Content -> Articles. Ensure each item in the dropdown functions correctly.

![thead-actions](https://cloud.githubusercontent.com/assets/2803503/26153697/7ddfc744-3b04-11e7-85a7-7c635f93e860.png)

### Documentation Changes Required
Yes
